solidity-v1/dashboard/src/pages/threshold/ThresholdUpgradePage.jsx

```diff
@@ -0,0 +1,320 @@
+              Random Beacon
+            </span>
+          </div>
+        </div>
+        {isDataFetching ? (
+          <TokenAmountSkeleton
+            wrapperClassName="staked__token-amount staked__token-amount--loading"
+            textStyles={{ width: "40%", marginLeft: ".3rem" }}
+            textClassName="h2"
+            icon={Icons.KeepOutline}
+            iconHeight={32}
+            iconWidth={32}
+            iconClassName="token-amount__icon keep-outline keep-outline--mint-80"
+          />
+        ) : (
+          <TokenAmount
+            wrapperClassName={"staked__token-amount mb-2"}
+            amount={stakedTotalAmount}
+            token={KEEP}
+            withIcon
+          />
+        )}
+        <AllocationProgressBar
+          title={"staked"}
+          currentValue={totalStakedAvailableKeep}
+          totalValue={stakedTotalAmount}
+          className={"mb-1"}
+          secondaryValue={totalStakedPendingKeep}
+          withLegend
+          currentValueLegendLabel={"Staked"}
+          secondaryValueLegendLabel={"Pending Undelegation"}
+          isDataFetching={isDataFetching}
+        />
+        <AllocationProgressBar
+          title={"undelegated"}
+          currentValue={totalUndelegatedAvailableKeep}
+          totalValue={stakedTotalAmount}
+          className={"mb-3"}
+          isDataFetching={isDataFetching}
+        />
+        <div className="upgrade-staked">
+          <h4 className={"mb-1"}>Upgrade Staked Tokens</h4>
+          <UpgradeTokensTile
+            title={"Staked"}
+            buttonDisabled={isDataFetching}
+            className={"mb-1"}
+            renderButton={() => (
+              <UpgradeTokensTile.NavLink
+                linkText={"undelegate"}
+                to={"/overview"}
+              />
+            )}
+          >
+            <UpgradeTokensTile.Row
+              label={"Total Pending KEEP"}
+              amount={totalStakedPendingKeep}
+              isDataFetching={isDataFetching}
+            />
+            <UpgradeTokensTile.Row
+              label={"Total Available KEEP"}
+              amount={totalStakedAvailableKeep}
+              isDataFetching={isDataFetching}
+            />
+          </UpgradeTokensTile>
+          <UpgradeTokensTile
+            title={"Undelegated"}
+            buttonDisabled={
+              lte(totalGrantedUnstakedBalance, 0) || isDataFetching
+            }
+            renderButton={() => (
+              <UpgradeTokensTile.NavLink
+                linkText={"claim tokens"}
+                to={"/overview"}
+              />
+            )}
+          >
+            <UpgradeTokensTile.Row
+              label={"Total Available KEEP"}
+              amount={totalUndelegatedAvailableKeep}
+              isDataFetching={isDataFetching}
+            />
+          </UpgradeTokensTile>
+        </div>
+      </section>
+    </section>
+  )
+}
+
+ThresholdUpgradePage.route = {
+  title: "Threshold Upgrade",
```

Comment:
Based on the Figma views it should be just `Upgrade`.

Reply:
Fixed in d298cb63a2ae9f64da792e0d13c963b4666af797